packages/atomic/create-atomic/src/plopfile.ts

```diff
@@ -51,72 +82,151 @@ export default function (plop: NodePlopAPI) {
         message:
           'The name of the security identity to impersonate, e.g. "alicesmith@example.com". See https://docs.coveo.com/en/56/#name-string-required.',
       },
-    ],
-    actions: [
-      {
-        type: 'add',
-        path: currentPath + '/{{project}}/.env',
-        templateFile: '../template/.env.hbs',
-      },
-      {
-        type: 'addMany',
-        destination: currentPath + '/{{project}}/',
-        base: '../template',
-        templateFiles: [
-          '../template/src/**',
-          '../template/scripts/*',
-          '../template/.env.example',
-          '../template/tsconfig.json',
-          '../template/stencil.config.ts',
-          '../template/netlify.toml',
-          '../template/README.md',
-          '../template/start-netlify.mjs',
-        ],
-      },
-      {
-        type: 'add',
-        path: currentPath + '/{{project}}/.gitignore',
-        templateFile: '../template/.gitignore.hbs',
-      },
       {
-        type: 'add',
-        path: currentPath + '/{{project}}/package.json',
-        templateFile: '../template/package.json.hbs',
-      },
-      function installPackagesPrompt() {
-        return 'Installing packages...';
-      },
-      function installPackages(answers) {
-        return new Promise((resolve, reject) => {
-          const {project} = answers as PromptsAnswers;
-          const process = spawn(getPackageManager(), ['install'], {
-            stdio: 'ignore',
-            cwd: `${currentPath}/${project}/`,
-          });
+        // Custom type necessary to allow bypassing async choices
```

Comment:
From what I saw, plop is really taking and reexposing inquirer typings.
Are you sure the promises are properly awaited?

Reply:
Yes it's properly awaited since I can see the list of pages that are fetched asynchronously. 

In the first comment you can see the (bad) validation  plop does to prevent me from using this, even if it works, since plop uses inquirer.